src/lib/components/Dropdown/Dropdown.tsx

```diff
@@ -35,6 +35,10 @@ export interface DropdownProps extends PropsWithChildren<Pick<FloatingProps, 'pl
   dismissOnClick?: boolean;
 }
 
+export interface TriggerWrapperProps extends ButtonProps {
+  getWidth?: (width: number) => void;
```

Comment:
Why don't we just refer to this as `setButtonWidth`, since that's how it is called when actually used on L127? Also, it's personal preference, but you can type this as `Dispatch<SetStateAction<number>>` to indicate that its specific purpose is as a React setState function.